index.html

```diff
@@ -1,48 +1,316 @@
+                        <li><button class="languages"> HTML</button></li>
+                        </ul>
+                </div>
+
+                
+                <div class="item6"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works5" class="container7">
+
+
+                <div class="item7"> <img alt="work5" src="img/work5.png"> </div>
+
+                    
+                                
+                <div class="item7"> <h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item7 btn-group">
+                    <ul id="btn5">
+                        <li><button class="languages"> Ruby on rails</button></li>
+                        <li><button class="languages"> CSS</button></li>
+                        <li><button class="languages"> JavaScript</button></li> 
+                        <li><button class="languages"> HTML</button></li>
+                        </ul>
+                </div>
+
+                
+                <div class="item7"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works6" class="container8">
+
+
+                <div class="item8"> <img alt="work6" src="img/work6.jpg"> </div>
+
+                    
+                                
+                <div class="item8"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item8 btn-group">
+                        <ul id="btn6">
+                            <li><button class="languages"> Ruby on rails</button></li>
+                            <li><button class="languages"> CSS</button></li>
+                            <li><button class="languages"> JavaScript</button></li> 
+                            <li><button class="languages"> HTML</button></li>
+                            </ul>
+                    </div>
+
+                    
+                    <div class="item8"><button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+    </div>
+        
+    
+        <section id="aboutme" class="container9">
+
+                <div class="about item9"><h1>About me</h1> </div>
+
+                <div class="description item9"><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have
+                    a project you need coded, don’t hestiate to contact me. </p>
+                </div>
+
+                <div class="item9">
+                    <button class="resume">My Resume</button>
+                </div>
+
+                <section class="item9 coding-languages">
+
+                    <div><img class="lang-icon" alt="lang icon" src="img/icon-languages.png"></div>
```

Comment:
- [ ] **OPTIONAL:** Please make these icons much bigger, and add some margin or padding at the top of it, this will make your portfolio UI/UX better.